app/components/SongCard/index.js

```diff
@@ -90,44 +87,46 @@ const StyledImage = styled.img`
   margin-bottom: 2em;
 `;
 
+const StyledPlayIcon = styled(PlayCircleTwoTone)`
+  && {
+    font-size: 18;
+  }
+`;
+const StyledStopIcon = styled(StopTwoTone)`
+  && {
+    font-size: 18;
```

Comment:
Here also

Reply:
by default it takes px right, so was considering this

Reply:
Changed accordingly